README.md

```diff
@@ -47,8 +47,22 @@ it.
 Run `make help` to see all the commands that are available for running the tests,
 linting, code formatting, etc.
 
-Via HTML's architecture
------------------------
+Configuration
+-------------
+
+Environment variables:
+
+| Name | Effect | Example |
+|------|--------|---------|
+| `VIA_BLOCKLIST_URL`   | Where to download the blocklist online | `https://hypothesis-via.s3-us-west-1.amazonaws.com/via-blocklist.txt` |
```

Comment:
Do we want to reveal this URL publicly? Especially since it's full of links to dirty sites

Reply:
Yeah... probably not